include/mbedtls/asn1write.h

```diff
@@ -381,6 +362,31 @@ mbedtls_asn1_named_data *mbedtls_asn1_store_named_data(mbedtls_asn1_named_data *
                                                        const unsigned char *val,
                                                        size_t val_len);
 
+/**
+ * \brief                  Encode an integer into ASN1 and write it to a buffer.
+ *                         The function takes a big endian bignum buffer that
+ *                         begins at start and ends at *p and encodes it into
+ *                         a asn1 frame.
+ *                         Warning: Destructive behaviour for the buffer data!
+ *
+ * \param p                The reference to the current position pointer.
+ * \param start            The start of the buffer, for bounds-checking.
+ * \param integer          The start of the integer buffer.
```

Comment:
It would be nice to recall the buffer representation here. In fact, if it's only in one place, I think it makes more sense for it to be in the parameter description than in the function's general description.